docs/installation/azuread.asciidoc

```diff
@@ -79,3 +80,48 @@ You can perform 802.1X authentication of users using Azure AD but this will only
 . Restart radiusd using `/usr/local/pf/bin/pfcmd service radiusd restart`
 . All the users matching this realm will now authenticate against Azure AD. Make sure you also have a connection profile with auto-registration enabled and the Azure AD source in it so that your users are correclty authorized when connecting.
 
+===== Using Azure AD EAP-TLS machine authentication
+
+You can perform a EAP-TLS authentication and verify the machine group membership in order to provide a access to the network.
+
+To do that first you will have to provide to the end device a certificate that contains the Device ID, to do this go in the Intune management interface and configure the template like this:
+
+image::Intune_SCEP_machine.png[scaledwidth="100%",alt="Intune SCEP machine configuration"]
+
+As you can see the CN (Common Name) will contain the Device Identifier, so when the device will connect on the secure ssid, the username will be equal to the device ID (like 8df07f7e-d98e-4579-aa97-bfcfaaa7fe38)
+
+Now it just a matter to retrieve the groupmembership associated with the device ID, in order to do that you will need to change the "User Groups Url" parameter in the "Azure Active Directory" authentication source.
+
+The URL will be `https://graph.microsoft.com/v1.0/devices(deviceId='%USERNAME')/memberOf` (for more information https://learn.microsoft.com/en-us/graph/api/device-list-memberof?view=graph-rest-1.0&tabs=http)
+
+In this example the api call will be `https://graph.microsoft.com/v1.0//devices(deviceId='8df07f7e-d98e-4579-aa97-bfcfaaa7fe38')/memberOf?$select=id,displayName and the reply is:
+
+`{
+   "value""=>"[
+      {
+         "displayName""=>""ZaymLed-Devices",
+         "id""=>""5c5f932c-08d4-46c3-bd93-11807f80ae35",
+         "@odata.type""=>""#microsoft.graph.group"
+      },
+      {
+         "id""=>""6ae04238-8e95-4f1b-8088-17c0d6cfbd98",
+         "displayName""=>""3D printer Laptop manager",
+         "@odata.type""=>""#microsoft.graph.group"
+      },
+      {
+         "@odata.type""=>""#microsoft.graph.group",
+         "id""=>""c2e304d6-f245-4ab2-8f60-58d78e57c526",
+         "displayName""=>""Windows 11 Feature Updates"
+      },
+      {
+         "@odata.type""=>""#microsoft.graph.group",
+         "id""=>""1bced11e-0cae-4689-8afa-060ec0b3341f",
+         "displayName""=>""ZaymLed - 3D software"
+      },
+   ],
+   "@odata.context""=>""https://graph.microsoft.com/v1.0/$metadata#directoryObjects(id,displayName)"
+}`
+
+In the "Azure Active Directory" authentication source create a authentication rule like this:
```

Comment:
"In the "Azure Active Directory" authentication source create a authentication rule like this:
" -> "From the "Azure Active Directory" authentication source, create an authentication rule like this:"